core/domain/exp_domain.py

```diff
@@ -367,49 +370,48 @@ def validate(self):
             raise utils.ValidationError(
                 'Expected math_images_generation_required to be an bool, '
                 'received %s' % self.math_images_generation_required)
-        if not isinstance(self.latex_values_without_svg, list):
+        if not isinstance(self.latex_strings_without_svg, list):
             raise utils.ValidationError(
-                'Expected latex_values to be a list, received %s' % (
-                    self.latex_values_without_svg))
-        for latex_value in self.latex_values_without_svg:
-            if not isinstance(latex_value, python_utils.BASESTRING):
+                'Expected latex_strings to be a list, received %s' % (
+                    self.latex_strings_without_svg))
+        for latex_string in self.latex_strings_without_svg:
+            if not isinstance(latex_string, python_utils.BASESTRING):
                 raise utils.ValidationError(
-                    'Expected each element in the list of latex values to be'
-                    ' a str, received %s' % latex_value)
+                    'Expected each element in the list of latex strings to be'
+                    ' a str, received %s' % latex_string)
 
     def get_svg_size_in_bytes(self):
-        """Returns the approximate size of SVG images for the latex values in
+        """Returns the approximate size of SVG images for the LaTeX strings in
         bytes.
 
         Returns:
             int. The approximate size of Math SVGs in bytes.
         """
 
-        # The approximate size for an SVG image for a Latex with one character
-        # is around 1000 Kb. But, when the number of characters increase the
+        # The approximate size for an SVG image for a LaTeX with one character
```

Comment:
LaTeX expression

(I mentioned this in a previous review comment https://github.com/oppia/oppia/pull/9889#discussion_r457262557)

Reply:
done